sample-tests/src/test/groovy/org/gradle/testretry/samples/PlaceholderAssertionErrorOutputNormalizer.groovy

```diff
@@ -7,7 +7,7 @@ class PlaceholderAssertionErrorOutputNormalizer implements OutputNormalizer {
     @Override
     String normalize(String commandOutput, ExecutionMetadata executionMetadata) {
         def result = commandOutput.split("\\r?\\n").collect { line ->
-            line.contains("org.gradle.internal.serialize.PlaceholderAssertionError") ? line.replace("org.gradle.internal.serialize.PlaceholderAssertionError", "PlaceholderAssertionError") : line
+            line.contains('org.gradle.internal.serialize.PlaceholderAssertionError') ? line.replace('org.gradle.internal.serialize.PlaceholderAssertionError', 'org.opentest4j.AssertionFailedError') : line
```

Comment:
I think we should link to https://github.com/gradle/gradle/pull/13995 here so we remember why this is here... 😉